### ISSUE TITLE
[BUG] 'all' selector is broken

### ISSUE THREAD

**Issue by temuri416** [alexandernst/jstree-actions#10]
Hi,

After updating jstree-actions to latest version with jsTree v3.3.4, the "all" selector stopped working:

```js
container.jstree(true).add_action("all", {
	"id": "action_remove",
	"class": "action_remove pull-right",
	"text": "",
	"after": true,
	"selector": "a",
	"event": "click",
	"callback": function(node_id, node, action_id, action_el){
		console.log("callback", node_id, action_id);
	}
});
```

Thanks!

**Comment by alexandernst** [comment]
Hi! Can you provide a small example in jsfiddle (or similar)?

**Comment by temuri416** [comment]
Sure, here it is:

https://codepen.io/tmrczk/pen/QOpReJ

Works with `https://rawgit.com/alexandernst/jstree-actions/0.1.2/jstree-actions.js`, but fails with `https://rawgit.com/alexandernst/jstree-actions/master/jstree-actions.js`.

**Comment by alexandernst** [comment]
Hmmm. @orangejenny this seems to be caused by your last PR. We both failed at realising that "node_id" can be the string "all". I fixed the bug and released 0.1.4. Can you both test that and check if everything works fine for you?
